project/plugins.sbt

```diff
@@ -12,6 +12,8 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.3.0")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.6.0")
 
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.7.5")
```

Comment:
Just because it's not a required plugin for building the connector, we don't have to worry about the version changing, we won't override anyone's global plugins (like mine :) )